rocketmq-spring-boot-starter/src/main/java/org/apache/rocketmq/spring/starter/RocketMQTransactionAnnotationProcessor.java

```diff
@@ -0,0 +1,94 @@
+package org.apache.rocketmq.spring.starter;
+
+import lombok.extern.slf4j.Slf4j;
```

Comment:
why this informal dependency for slf4j?

Reply:
other old classes also use the this. i.e. DefaultRocketMQListenerContainer.class, RocketMQTemplate.class